### PR TITLE
ffmpeg6: wave2

### PR DESCRIPTION
**Pull request by zlice**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, **x86_64**

#### Comments

More media programs and games. synfig (and synfigstudio) use `mlt7` which means `mlt` can be removed.  `xine-lib` and `osg` are used by `openmw` and `sumo`.

### CODE REVIEW

File: srcpkgs/removed-packages/template [hunk 1]
```diff
@@ -454,6 +454,7 @@ replaces="
  mimms<=3.2.1_4
  miro-video-converter<=3.0.2_3
  mirrorbits<=0.5.1_1
+ mlt<=6.26.1_3
```
Comment (classabbyamp) [inline srcpkgs/removed-packages/template:457]: missing mlt subpackages

File: srcpkgs/removed-packages/template [hunk 2]
```diff
@@ -454,6 +454,11 @@ replaces="
...
+ mlt-data<=6.26.1_3
+ mlt-devel<=6.26.1_3
+ mlt-doc<=6.26.1_3
+ mlt-python3<=6.26.1_3
```
Comment (classabbyamp) [inline srcpkgs/removed-packages/template:461]: missing mlt-python

**Comment by sgn** [comment]
I would rebuild all Qt5 related package in a single batch

**Comment by zlice** [comment]
openmw appears to be building fine with cmake-3.29.5

i see there were updates to 3.30.x in the past month. any advice?

```
2024-07-23T02:04:22.6966193Z This warning is for project developers.  Use -Wno-dev to suppress it.
2024-07-23T02:04:22.6966569Z 
2024-07-23T02:04:22.7447575Z -- Found Boost: /usr/include (found suitable version "1.83.0", minimum required is "1.6.2") found components: system filesystem program_options iostreams regex
2024-07-23T02:04:22.7455365Z CMake Warning (dev) at CMakeLists.txt:447 (find_package):
2024-07-23T02:04:22.7456343Z   Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
2024-07-23T02:04:22.7457603Z   --help-policy CMP0167" for policy details.  Use the cmake_policy command to
2024-07-23T02:04:22.7458479Z   set the policy and suppress this warning.
2024-07-23T02:04:22.7458915Z 
2024-07-23T02:04:22.7459408Z This warning is for project developers.  Use -Wno-dev to suppress it.
2024-07-23T02:04:22.7460035Z 
2024-07-23T02:04:22.7529079Z -- Found Boost: /usr/include (found suitable version "1.83.0", minimum required is "1.77.0") found components: atomic
2024-07-23T02:04:22.8372667Z -- Found MyGUI 3.4.2
2024-07-23T02:04:22.9202764Z -- Found SDL2 2.30.4
2024-07-23T02:04:22.9254146Z -- Found OpenAL: /usr/lib/libopenal.so
2024-07-23T02:04:22.9653446Z -- Found LuaJit 
2024-07-23T02:04:24.4015840Z -- LTO enabled for Release configuration.
2024-07-23T02:04:24.5174514Z -- Found SQLite3: /usr/include (found version "3.45.3")
2024-07-23T02:04:24.6030209Z -- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
2024-07-23T02:04:24.6045542Z -- Configuring done (4.3s)
2024-07-23T02:04:24.8417268Z CMake Error at components/CMakeLists.txt:405 (target_link_libraries):
2024-07-23T02:04:24.8418004Z   Target "components" links to:
2024-07-23T02:04:24.8418289Z 
2024-07-23T02:04:24.8418447Z     RecastNavigation::DebugUtils
2024-07-23T02:04:24.8418716Z 
2024-07-23T02:04:24.8418952Z   but the target was not found.  Possible reasons include:
2024-07-23T02:04:24.8419353Z 
2024-07-23T02:04:24.8419514Z     * There is a typo in the target name.
2024-07-23T02:04:24.8420030Z     * A find_package call is missing for an IMPORTED target.
2024-07-23T02:04:24.8420539Z     * An ALIAS target is missing.
```

**Comment by zlice** [comment]
> I would rebuild all Qt5 related package in a single batch

i was grouping different but that makes sense. the bottom set would be added. have to redo the other 'waves', maybe just axe 4 and put everything in 3

```
openshot
mgba
hedgewars
ppsspp
retroarch
openmw

goldendict
kpipewire
kf6-kfilemetadata
kfilemetadata5
ssr
QMPlay2
audacious-plugins
avidemux
qmmp
mixxx
cantata
```

**Comment by zlice** [comment]
added qt depending packages

**Comment by zlice** [comment]
i686 segfault actually looks like this is in `kf6-kcoreaddons-32bit-6.3.0_1`

```
Python Exception <class 'ModuleNotFoundError'>: No module named 'gdb'
/destdir/gdb-15.1/usr/bin/gdb: warning: 
Could not load the Python gdb module from `/usr/share/gdb/python'.
Limited Python support is available from the _gdb module.
Suggest passing --data-directory=/path/to/gdb/data-directory.
GNU gdb (GDB) 15.1
Copyright (C) 2024 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "i686-pc-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from builddir/kf6-kfilemetadata-6.3.0/./build/bin/extractorcollectiontest...
(gdb) r
Starting program: /builddir/kf6-kfilemetadata-6.3.0/build/bin/extractorcollectiontest 
Python Exception <class 'ModuleNotFoundError'>: No module named 'gdb'
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/usr/lib32/libthread_db.so.1".
********* Start testing of KFileMetaData::ExtractorCollectionTest *********
Config: Using QtTest library 6.7.2, Qt 6.7.2 (i386-little_endian-ilp32 shared (dynamic) release build; by GCC 13.2.0), void unknown
PASS   : KFileMetaData::ExtractorCollectionTest::initTestCase()

Program received signal SIGSEGV, Segmentation fault.
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
0xf726bcdb in ?? () from /usr/lib32/libKF6CoreAddons.so.6
(gdb) thread apply all bt

Thread 1 (Thread 0xf66f26c0 (LWP 27513) "extractorcollec"):
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
Python Exception <class 'ModuleNotFoundError'>: No module named 'gdb'
#0  0xf726bcdb in ?? () from /usr/lib32/libKF6CoreAddons.so.6
#1  0xf72c02e5 in ?? () from /usr/lib32/libKF6CoreAddons.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#2  0xf72bbf2d in KPluginMetaData::findPlugins(QString const&, std::function<bool (KPluginMetaData const&)>, QFlags<KPluginMetaData::KPluginMetaDataOption>) () from /usr/lib32/libKF6CoreAddons.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#3  0xf7f2d7fb in KFileMetaData::ExtractorCollectionPrivate::findExtractors() () from /builddir/kf6-kfilemetadata-6.3.0/build/bin/libKF6FileMetaData.so.3
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#4  0x5655bbe6 in KFileMetaData::ExtractorCollectionTest::testFetchExtractors() [clone .isra.0] ()
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#5  0xf7a74e8b in QMetaMethodInvoker::invokeImpl(QMetaMethod, void*, Qt::ConnectionType, int, void const* const*, char const* const*, QtPrivate::QMetaTypeInterface const* const*) () from /usr/lib32/libQt6Core.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#6  0xf7a75a55 in QMetaMethod::invokeImpl(QMetaMethod, void*, Qt::ConnectionType, int, void const* const*, char const* const*, QtPrivate::QMetaTypeInterface const* const*) () from /usr/lib32/libQt6Core.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#7  0xf7f72eb9 in ?? () from /usr/lib32/libQt6Test.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#8  0xf7f7ce59 in ?? () from /usr/lib32/libQt6Test.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#9  0xf7f7db75 in ?? () from /usr/lib32/libQt6Test.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#10 0xf7f7e49f in ?? () from /usr/lib32/libQt6Test.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#11 0xf7f7f00b in QTest::qRun() () from /usr/lib32/libQt6Test.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#12 0xf7f7f827 in QTest::qExec(QObject*, int, char**) () from /usr/lib32/libQt6Test.so.6
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
#13 0x565586f4 in main ()
Python Exception <class 'NameError'>: Installation error: gdb._execute_unwinders function is missing
(gdb) disas 0xf726bcdb
No function contains specified address.
(gdb) inf reg
eax            0x5656f4f0          1448539376
ecx            0x5656ef90          1448538000
edx            0x0                 0
ebx            0x0                 0
esp            0xffffd100          0xffffd100
ebp            0x5656f4fc          0x5656f4fc
esi            0x0                 0
edi            0x5656efa0          1448538016
eip            0xf726bcdb          0xf726bcdb
eflags         0x10293             [ CF AF SF IF RF ]
cs             0x23                35
ss             0x2b                43
ds             0x2b                43
es             0x2b                43
fs             0x0                 0
gs             0x63                99
(gdb) disas $eip
No function contains specified address.
```

**Comment by zlice** [comment]
revbump for kf6-kcoreaddons fixes kfilemetadata checks

updated avidemux and patched, can make separate PR if that matters

**Comment by zlice** [comment]
Added hedgewars patch/changes, assume cmake 3.30 issue?

Disabled i686 "x86 optimizations" (128bit xmm assembly)

**Comment by zlice** [comment]
Unless I'm going crazy...I'm pretty sure `ppsspp` i686 build fine the other day before qt/kf6 stuff updated. It doesn't look like any of its deps updated so I'm not sure why it can't pass test now.

**Comment by classabbyamp** [comment]
https://github.com/void-linux/void-packages/compare/d3845b26...9517cc4a

**Pull request closed without merging**